lib/MusicBrainz/Server/Controller/OAuth2.pm

```diff
@@ -36,6 +36,7 @@ sub authorize : Local Args(0) RequireAuth CSRFToken
     # https://tools.ietf.org/html/draft-ietf-oauth-security-topics-14#section-4.14
     # Authorization servers MUST prevent clickjacking attacks.
     $c->response->header('X-Frame-Options' => 'DENY');
+    $c->response->header('Content-Security-Policy' => q(default-src 'self' staticbrainz.org));
```

Comment:
Right, since I'm only setting it in `sub authorize` there. But I do set CSP headers on a ton more pages in the commit titled "Add CSP headers to other account forms." Which is mainly only on account/admin forms where CSRF tokens were added before, not edit pages.